package.json

```
@@ -32,11 +32,11 @@
   "dependencies": {
     "@sentry/browser": "4.6.2 || ~4.6.4",
     "@sentry/core": "4.6.2 || ~4.6.4",
-    "@sentry/node": "4.6.2 || ~4.6.4",
     "@sentry/minimal": "^4.5.0",
+    "@sentry/node": "4.6.2 || ~4.6.4",
     "@sentry/types": "^4.5.0",
     "@sentry/utils": "4.6.2 || ~4.6.4",
-    "electron-fetch": "1.1.0",
+    "electron-fetch": "^1.3.0",
```

```suggestion
    "electron-fetch": "1.3.0",
```
Please pin the dependency, aside from that 💯